pytest/functional/test_hydroshare_server.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+
+
+@pytest.mark.django_db
+def test_my_resources_page(admin_client, resource_with_metadata):
+    """Resource with unique id is found on my resources page"""
+    response = admin_client.get('/my-resources/', follow=True)
+    assert resource_with_metadata in response.content
```

Comment:
access My Resources and look for the title that contains the UUID that was generated in the fixture for the Resource Title, then passed as the function return to the test; a title with a unique UUID was created and found on the page when the admin_client visited My Resources (as the admin:default user)

Reply:
I was mostly not sure what this test was doing. Now I get it after reading your comments above. So mostly this test is UI testing?

Reply:
It's functional test, generally targeting REST calls, endpoints and or templates

Reply:
"UI" testing would be something like Jest, mocha, chai, jasmine, behave (python selenium)....